dev/com.ibm.ws.jpa.container.core/src/com/ibm/ws/jpa/management/JPAPUnitInfo.java

```diff
@@ -900,10 +900,8 @@ private EntityManagerFactory createEMFactory(PersistenceUnitInfo puInfo, boolean
         getJPAComponent().addIntegrationProperties(xmlSchemaVersion,
                                                    integrationProperties, ivClassLoader);
 
-        if (ProductInfo.getBetaEdition()) {
-            // Add default persistence properties supplied from JPAComponent configuration
-            getJPAComponent().addDefaultProperties(integrationProperties);
-        }
+        // Add default persistence properties supplied from JPAComponent configuration
```

Comment:
Please remove the now unused import of com.ibm.ws.kernel.productinfo.ProductInfo;